packages/ui/src/components/logo/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { siBitbucket, siGithub, siGitlab, siJira, siKubernetes, type SimpleIcon } from 'simple-icons'
+
+const LogoSet = [siBitbucket, siGithub, siGitlab, siJira, siKubernetes]
+
+/**
+ * Generates a map of logo names to their corresponding `SimpleIcon` data.
+ *
+ * @returns {Map<string, SimpleIcon>} A map where the keys are logo slugs (e.g., `"github"`)
+ * and the values are `SimpleIcon` objects containing path data and brand color.
+ *
+ * @example
+ * ```ts
+ * const logoMap = getLogoNameMap();
+ * const githubIcon = logoMap.get("github");
+ * console.log(githubIcon?.path); // SVG path data
+ * ```
+ */
+export const getLogoNameMap = (): Map<string, SimpleIcon> =>
```

Comment:
ooh.. i think there was some misunderstanding.
this logic should NOT be in the logo component or in the `ui` package. the earlier simple implementation was correct.
instead, the usage of it should be via a map in the view layer.

Reply:
Done, removed this logic.